asmcli/asmcli.sh

```diff
@@ -922,10 +932,19 @@ EOF
     fi
   fi
 
-  # GCP only: when no Fleet Id is provided, default to the cluster's project as the Fleet host.
-  if [[ -z "${FLEET_ID}" ]] && is_gcp; then
-    FLEET_ID="${PROJECT_ID}"
-    context_set-option "FLEET_ID" "${FLEET_ID}"
+  if is_gcp; then
+    # GCP only: when no Fleet Id is provided, default to the cluster's project as the Fleet host.
+    if [[ -z "${FLEET_ID}" ]]; then
+      FLEET_ID="${PROJECT_ID}"
+      context_set-option "FLEET_ID" "${FLEET_ID}"
+    fi
+  else
+    # non-GCP only: set Project Id to same as Fleet Id.
```

Comment:
nit: I don't think we need "GCP only" and "non-GCP" right below the if statement that checks whether it's GCP or not.

Reply:
assuming that this is about the comment being redundant - i will fix that.